src/actions/pause/paused.js

```diff
@@ -44,8 +46,13 @@ export function paused(pauseInfo: Pause) {
   return async function({ dispatch, getState, client, sourceMaps }: ThunkArgs) {
     const { frames, why, loadedObjects } = pauseInfo;
     const rootFrame = frames.length > 0 ? frames[0] : null;
+    let reactComponentStack;
 
     if (rootFrame) {
+      if (isReactComponent(rootFrame.this)) {
+        reactComponentStack = await getReactComponentStack(rootFrame, client);
+      }
```

Comment:
i think we should try to put this in the `extra` field. that way we have one place for react stuff

Reply:
will do